src/renderer/components/app.tsx

```diff
@@ -152,38 +148,6 @@ export class App extends React.Component {
     });
   }
 
-  @observable extensionRoutes: Map<ClusterPageMenuRegistration, React.ReactNode> = new Map();
-
-  generateExtensionTabLayoutRoutes(rootItems: ClusterPageMenuRegistration[]) {
```

Comment:
What was this used for previously? It isn't being used anywhere.

Reply:
Wasn't it called on line 95?

Reply:
It was called yes but the field `extensionRoutes` was never read from outside of that function

Reply:
True. `extensionRoutes` was used for some time, but after [this changes](https://github.com/lensapp/lens/pull/1990/files#diff-bf0690965bbc846224b78fb77bb20125b2c02c3e48553d6c35f825d7b20bd01e) no longer in use